src/main/webapp/dev/js/main/errorPageEmailComposer.es6

```diff
@@ -0,0 +1,54 @@
+import {
+    clearStatusMessages,
+    setStatusMessageToForm,
+} from '../common/statusMessage.es6';
+
+import {
+    StatusType,
+} from '../common/const.es6';
+
+const ERROR_STATUS_MESSAGE = 'Failed to record the error message. Please email our support team at '
+        + '<a href="mailto:teammates@comp.nus.edu.sg">teammates@comp.nus.edu.sg</a>.';
```

Comment:
It might be better to put a constant for the email address in `const.es6`, and to use [template literals](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals) for this string.